contracts/consensus/CoconsensusModule.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "./CoconsensusI.sol";
+
+contract CoconsensusModule {
+
+    /** Address of Coconsensus contract on auxiliary chain. */
+    address private  coconsensusAddress = address(0x0000000000000000000000000000000000004D00);
```

Comment:
```suggestion
    address private constant  coconsensusAddress = address(0x0000000000000000000000000000000000004D00);
```

Reply:
Resolving after discussion.

Reply:
It has been purposely removed. 
With constant function `getCoconsensus()` is `pure`, and when we override it in the tests it has to be a `view` function, the compiler complains about it. 

We are achieving our objective without it being a constant.